src/app/survey/survey.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, ParamMap } from '@angular/router';
+<<<<<<< HEAD
```

Comment:
merge conflicts in this file, but this might should be completely removed anyway.  Make the changes here (if any) to `app/review.review.component.ts`

Reply:
I merged the conflicts: When I build I get this: I think it is because I don't have the right dependencies:

ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/metadata.d.ts (1,212): Cannot find module './metadata/di'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/metadata.d.ts (2,126): Cannot find module './metadata/directives'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/metadata.d.ts (3,127): Cannot find module './metadata/lifecycle_hooks'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/metadata.d.ts (4,105): Cannot find module './metadata/ng_module'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/metadata.d.ts (5,35): Cannot find module './metadata/view'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/di.d.ts (13,15): Cannot find module './di/metadata'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/di.d.ts (14,61): Cannot find module './di/forward_ref'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/di.d.ts (15,26): Cannot find module './di/injector'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/di.d.ts (16,36): Cannot find module './di/reflective_injector'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/di.d.ts (17,105): Cannot find module './di/provider'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/di.d.ts (18,71): Cannot find module './di/reflective_provider'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/di.d.ts (19,31): Cannot find module './di/reflective_key'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/di.d.ts (20,45): Cannot find module './di/injection_token'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/application_ref.d.ts (13,33): Cannot find module './linker/compiler'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/application_ref.d.ts (14,48): Cannot find module './linker/component_factory'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/application_ref.d.ts (15,42): Cannot find module './linker/component_factory_resolver'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/application_ref.d.ts (16,46): Cannot find module './linker/ng_module_factory'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/application_ref.d.ts (17,25): Cannot find module './linker/view_ref'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/application_tokens.d.ts (9,30): Cannot find module './linker/component_factory'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/render/api.d.ts (9,35): Cannot find module '../metadata/view'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/linker.d.ts (8,108): Cannot find module './linker/compiler'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/linker.d.ts (9,48): Cannot find module './linker/component_factory'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/linker.d.ts (10,42): Cannot find module './linker/component_factory_resolver'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/linker.d.ts (11,28): Cannot find module './linker/element_ref'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/linker.d.ts (12,46): Cannot find module './linker/ng_module_factory'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/linker.d.ts (13,57): Cannot find module './linker/ng_module_factory_loader'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/linker.d.ts (14,27): Cannot find module './linker/query_list'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/linker.d.ts (15,70): Cannot find module './linker/system_js_ng_module_factory_loader'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/linker.d.ts (16,29): Cannot find module './linker/template_ref'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/linker.d.ts (17,34): Cannot find module './linker/view_container_ref'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/linker.d.ts (18,42): Cannot find module './linker/view_ref'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/debug/debug_node.d.ts (9,30): Cannot find module '../view/index'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/change_detection.d.ts (13,395): Cannot find module './change_detection/change_detection'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/i18n/tokens.d.ts (8,32): Cannot find module '../di/injection_token'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/application_module.d.ts (2,50): Cannot find module './change_detection/change_detection'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/errors.d.ts (8,30): Cannot find module './view'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/core_private_export.d.ts (10,82): Cannot find module './change_detection/change_detection_util'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/core_private_export.d.ts (11,59): Cannot find module './change_detection/change_detection_util'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/core_private_export.d.ts (12,134): Cannot find module './change_detection/constants'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/core_private_export.d.ts (15,55): Cannot find module './linker/component_factory'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/core_private_export.d.ts (16,85): Cannot find module './linker/component_factory_resolver'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/core_private_export.d.ts (17,47): Cannot find module './metadata/view'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/core_private_export.d.ts (18,67): Cannot find module './reflection/reflection_capabilities'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/core_private_export.d.ts (19,85): Cannot find module './reflection/types'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/core_private_export.d.ts (22,49): Cannot find module './util/decorators'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/core_private_export.d.ts (23,72): Cannot find module './util/lang'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/core_private_export.d.ts (24,106): Cannot find module './view/index'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/core_private_export.d.ts (25,97): Cannot find module './view/provider'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/codegen_private_exports.d.ts (8,85): Cannot find module './linker/component_factory_resolver'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/codegen_private_exports.d.ts (9,65): Cannot find module './linker/ng_module_factory_loader'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/codegen_private_exports.d.ts (10,920): Cannot find module './view/index'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/core.d.ts (15,55): Cannot find module './util/decorators'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/core.d.ts (29,90): Cannot find module './profile/profile'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/src/core.d.ts (36,15): Cannot find module './animation/animation_metadata_wrapped'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/core.d.ts (5,169): Cannot find module './src/animation/dsl'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/core.d.ts (9,76): Cannot find module './src/change_detection/change_detection'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/core.d.ts (10,52): Cannot find module './src/change_detection/differs/default_iterable_differ'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/core.d.ts (11,52): Cannot find module './src/change_detection/differs/default_keyvalue_differ'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/core.d.ts (12,43): Cannot find module './src/di/reflective_injector'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/core.d.ts (13,78): Cannot find module './src/di/reflective_provider'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/core.d.ts (14,34): Cannot find module './src/profile/profile'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/core.d.ts (15,107): Cannot find module './src/profile/wtf_impl'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/core.d.ts (16,42): Cannot find module './src/util/decorators'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/core.d.ts (17,28): Cannot find module './src/view/provider'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/core/core.d.ts (18,36): Cannot find module './src/view/types'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/cdk/a11y/typings/aria-describer.d.ts (8,10): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Optional'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/cdk/a11y/typings/live-announcer.d.ts (8,26): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Optional'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/cdk/a11y/typings/focus-monitor.d.ts (8,55): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Optional'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/cdk/bidi/typings/directionality.d.ts (8,24): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Optional'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/material/core/typings/coordination/unique-selection-dispatcher.d.ts (8,10): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Optional'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/cdk/scrolling/typings/scroll-dispatcher.d.ts (8,30): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Optional'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/cdk/scrolling/typings/viewport-ruler.d.ts (8,10): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Optional'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/cdk/overlay/typings/overlay-container.d.ts (8,10): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Optional'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/material/icon/typings/icon-registry.d.ts (8,10): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Optional'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/animations/browser.d.ts (6,16): Cannot find module './browser/index'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/platform-browser/animations/src/animation_renderer.d.ts (1,10): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/animations/browser"' has no exported member 'ɵAnimationEngine'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/platform-browser/animations/src/providers.d.ts (1,10): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/animations/browser"' has no exported member 'AnimationDriver'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/platform-browser/animations/src/providers.d.ts (1,27): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/animations/browser"' has no exported member 'ɵAnimationEngine'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/platform-browser/animations/src/providers.d.ts (1,64): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/animations/browser"' has no exported member 'ɵAnimationStyleNormalizer'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/platform-browser/animations/src/providers.d.ts (1,119): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/animations/browser"' has no exported member 'ɵNoopAnimationDriver'.
ERROR in C:/Users/Ciara/Rater-System/node_modules/@angular/platform-browser/animations/src/providers.d.ts (1,164): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/animations/browser"' has no exported member 'ɵWebAnimationsStyleNormalizer'.
ERROR in C:/Users/Ciara/Rater-System/src/app/app.component.ts (8,10): Duplicate identifier 'Student'.
ERROR in C:/Users/Ciara/Rater-System/src/app/app.component.ts (13,10): Duplicate identifier 'Student'.
ERROR in C:/Users/Ciara/Rater-System/src/app/app.component.ts (9,10): Duplicate identifier 'Teacher'.
ERROR in C:/Users/Ciara/Rater-System/src/app/app.component.ts (14,10): Duplicate identifier 'Teacher'.
ERROR in C:/Users/Ciara/Rater-System/src/app/app.component.ts (7,29): Cannot find module './data.service'.
ERROR in C:/Users/Ciara/Rater-System/src/app/app.component.ts (8,25): Cannot find module './student'.
ERROR in C:/Users/Ciara/Rater-System/src/app/app.component.ts (9,25): Cannot find module './teacher-detail/teacher'.
ERROR in C:/Users/Ciara/Rater-System/src/app/app.component.ts (10,32): Cannot find module './teacher-detail/teacher.service'.
ERROR in C:/Users/Ciara/Rater-System/src/app/teacher/teacher.service.ts (1,10): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Injectable'.
ERROR in C:/Users/Ciara/Rater-System/src/app/review/review.service.ts (1,10): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Injectable'.
ERROR in C:/Users/Ciara/Rater-System/src/app/course/course.service.ts (1,10): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Injectable'.
ERROR in C:/Users/Ciara/Rater-System/src/app/student/student.service.ts (1,10): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Injectable'.
ERROR in C:/Users/Ciara/Rater-System/src/app/administrator/administrator.service.ts (1,10): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Injectable'.
ERROR in C:/Users/Ciara/Rater-System/src/app/survey/survey.component.ts (3,1): Merge conflict marker encountered.
ERROR in C:/Users/Ciara/Rater-System/src/app/survey/survey.component.ts (7,1): Merge conflict marker encountered.
ERROR in C:/Users/Ciara/Rater-System/src/app/survey/survey.component.ts (9,1): Merge conflict marker encountered.
ERROR in C:/Users/Ciara/Rater-System/src/app/survey/survey.component.ts (20,1): Merge conflict marker encountered.
ERROR in C:/Users/Ciara/Rater-System/src/app/survey/survey.component.ts (28,1): Merge conflict marker encountered.
ERROR in C:/Users/Ciara/Rater-System/src/app/survey/survey.component.ts (36,1): Merge conflict marker encountered.
ERROR in C:/Users/Ciara/Rater-System/src/app/survey/survey.component.ts (41,1): Merge conflict marker encountered.
ERROR in C:/Users/Ciara/Rater-System/src/app/survey/survey.component.ts (43,1): Merge conflict marker encountered.
ERROR in C:/Users/Ciara/Rater-System/src/app/survey/survey.component.ts (45,1): Merge conflict marker encountered.
ERROR in C:/Users/Ciara/Rater-System/src/app/survey/survey.component.ts (11,25): Cannot find module '../teacher-detail/teacher'.
ERROR in C:/Users/Ciara/Rater-System/src/app/survey/survey.component.ts (12,32): Cannot find module '../teacher-detail/teacher.service'.
ERROR in Couldn't resolve original symbol for ./src/util/decorators from C:/Users/Ciara/Rater-System/node_modules/@angular/core/core.d.ts
ERROR in C:/Users/Ciara/Rater-System/src/app/app.component.ts (8,10): Duplicate identifier 'Student'.
ERROR in C:/Users/Ciara/Rater-System/src/app/app.component.ts (13,10): Duplicate identifier 'Student'.
ERROR in C:/Users/Ciara/Rater-System/src/app/app.component.ts (9,10): Duplicate identifier 'Teacher'.
ERROR in C:/Users/Ciara/Rater-System/src/app/app.component.ts (14,10): Duplicate identifier 'Teacher'.
ERROR in C:/Users/Ciara/Rater-System/src/app/app.component.ts (7,29): Cannot find module './data.service'.
ERROR in C:/Users/Ciara/Rater-System/src/app/app.component.ts (8,25): Cannot find module './student'.
ERROR in C:/Users/Ciara/Rater-System/src/app/app.component.ts (9,25): Cannot find module './teacher-detail/teacher'.
ERROR in C:/Users/Ciara/Rater-System/src/app/app.component.ts (10,32): Cannot find module './teacher-detail/teacher.service'.
ERROR in C:/Users/Ciara/Rater-System/src/app/review/review.service.ts (1,10): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Injectable'.
ERROR in C:/Users/Ciara/Rater-System/src/app/course/course.service.ts (1,10): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Injectable'.
ERROR in C:/Users/Ciara/Rater-System/src/app/student/student.service.ts (1,10): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Injectable'.
ERROR in C:/Users/Ciara/Rater-System/src/app/teacher/teacher.service.ts (1,10): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Injectable'.
ERROR in C:/Users/Ciara/Rater-System/src/app/administrator/administrator.service.ts (1,10): Module '"C:/Users/Ciara/Rater-System/node_modules/@angular/core/core"' has no exported member 'Injectable'.
ERROR in ./node_modules/@angular/platform-browser/@angular/platform-browser/animations.es5.js
Module not found: Error: Can't resolve '@angular/animations/browser' in 'C:\Users\Ciara\Rater-System\node_modules\@angular\platform-browser\@angular\platform-browser'
resolve '@angular/animations/browser' in 'C:\Users\Ciara\Rater-System\node_modules\@angular\platform-browser\@angular\platform-browser'
  Parsed request is a module
  using description file: C:\Users\Ciara\Rater-System\node_modules\@angular\platform-browser\package.json (relative path: ./@angular/platform-browser)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: C:\Users\Ciara\Rater-System\node_modules\@angular\platform-browser\package.json (relative path: ./@angular/platform-browser)
    resolve as module
      C:\Users\Ciara\Rater-System\node_modules\@angular\platform-browser\@angular\platform-browser\node_modules doesn't exist or is not a directory
      C:\Users\Ciara\Rater-System\node_modules\@angular\platform-browser\@angular\node_modules doesn't exist or is not a directory
      C:\Users\Ciara\Rater-System\node_modules\@angular\platform-browser\node_modules doesn't exist or is not a directory
      C:\Users\Ciara\Rater-System\node_modules\@angular\node_modules doesn't exist or is not a directory
      C:\Users\Ciara\Rater-System\node_modules\node_modules doesn't exist or is not a directory
      C:\Users\node_modules doesn't exist or is not a directory
      C:\node_modules doesn't exist or is not a directory
      looking for modules in C:\Users\Ciara\Rater-System\node_modules
        using description file: C:\Users\Ciara\Rater-System\package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: C:\Users\Ciara\Rater-System\package.json (relative path: ./node_modules)
          using description file: C:\Users\Ciara\Rater-System\node_modules\@angular\animations\package.json (relative path: ./browser)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Ciara\Rater-System\node_modules\@angular\animations\browser doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Ciara\Rater-System\node_modules\@angular\animations\browser.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Ciara\Rater-System\node_modules\@angular\animations\browser.js doesn't exist
            as directory
              C:\Users\Ciara\Rater-System\node_modules\@angular\animations\browser doesn't exist
      looking for modules in C:\Users\Ciara\node_modules
        No description file found
        Field 'browser' doesn't contain a valid alias configuration
        No description file found
        no extension
          Field 'browser' doesn't contain a valid alias configuration
          C:\Users\Ciara\node_modules\@angular\animations\browser doesn't exist
        .ts
          Field 'browser' doesn't contain a valid alias configuration
          C:\Users\Ciara\node_modules\@angular\animations\browser.ts doesn't exist
        .js
          Field 'browser' doesn't contain a valid alias configuration
          C:\Users\Ciara\node_modules\@angular\animations\browser.js doesn't exist
        as directory
          C:\Users\Ciara\node_modules\@angular\animations\browser doesn't exist
      looking for modules in C:\Users\Ciara\Rater-System\node_modules
        using description file: C:\Users\Ciara\Rater-System\package.json (relative path: ./node_modules)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: C:\Users\Ciara\Rater-System\package.json (relative path: ./node_modules)
          using description file: C:\Users\Ciara\Rater-System\node_modules\@angular\animations\package.json (relative path: ./browser)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Ciara\Rater-System\node_modules\@angular\animations\browser doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Ciara\Rater-System\node_modules\@angular\animations\browser.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Ciara\Rater-System\node_modules\@angular\animations\browser.js doesn't exist
            as directory
              C:\Users\Ciara\Rater-System\node_modules\@angular\animations\browser doesn't exist
      looking for modules in C:\Users\Ciara\Rater-System\src
        using description file: C:\Users\Ciara\Rater-System\package.json (relative path: ./src)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: C:\Users\Ciara\Rater-System\package.json (relative path: ./src)
          using description file: C:\Users\Ciara\Rater-System\package.json (relative path: ./src/@angular/animations/browser)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Ciara\Rater-System\src\@angular\animations\browser doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Ciara\Rater-System\src\@angular\animations\browser.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Ciara\Rater-System\src\@angular\animations\browser.js doesn't exist
            as directory
              C:\Users\Ciara\Rater-System\src\@angular\animations\browser doesn't exist
      looking for modules in C:\Users\Ciara\Rater-System\src
        using description file: C:\Users\Ciara\Rater-System\package.json (relative path: ./src)
          Field 'browser' doesn't contain a valid alias configuration
        after using description file: C:\Users\Ciara\Rater-System\package.json (relative path: ./src)
          using description file: C:\Users\Ciara\Rater-System\package.json (relative path: ./src/@angular/animations/browser)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Ciara\Rater-System\src\@angular\animations\browser doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Ciara\Rater-System\src\@angular\animations\browser.ts doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\Ciara\Rater-System\src\@angular\animations\browser.js doesn't exist
            as directory
              C:\Users\Ciara\Rater-System\src\@angular\animations\browser doesn't exist
[C:\Users\Ciara\Rater-System\node_modules\@angular\platform-browser\@angular\platform-browser\node_modules]
[C:\Users\Ciara\Rater-System\node_modules\@angular\platform-browser\@angular\node_modules]
[C:\Users\Ciara\Rater-System\node_modules\@angular\platform-browser\node_modules]
[C:\Users\Ciara\Rater-System\node_modules\@angular\node_modules]
[C:\Users\Ciara\Rater-System\node_modules\node_modules]
[C:\Users\node_modules]
[C:\node_modules]
[C:\Users\Ciara\node_modules\package.json]
[C:\Users\Ciara\Rater-System\node_modules\@angular\animations\browser]
[C:\Users\Ciara\Rater-System\node_modules\@angular\animations\browser.ts]
[C:\Users\Ciara\Rater-System\node_modules\@angular\animations\browser.js]
[C:\Users\Ciara\Rater-System\node_modules\@angular\animations\browser]
[C:\Users\Ciara\node_modules\@angular\animations\browser\package.json]
[C:\Users\Ciara\node_modules\@angular\animations\browser]
[C:\Users\Ciara\node_modules\@angular\animations\browser.ts]
[C:\Users\Ciara\node_modules\@angular\animations\browser.js]
[C:\Users\Ciara\node_modules\@angular\animations\browser]
[C:\Users\Ciara\Rater-System\node_modules\@angular\animations\browser]
[C:\Users\Ciara\Rater-System\node_modules\@angular\animations\browser.ts]
[C:\Users\Ciara\Rater-System\node_modules\@angular\animations\browser.js]
[C:\Users\Ciara\Rater-System\node_modules\@angular\animations\browser]
[C:\Users\Ciara\Rater-System\src\@angular\animations\browser]
[C:\Users\Ciara\Rater-System\src\@angular\animations\browser.ts]
[C:\Users\Ciara\Rater-System\src\@angular\animations\browser.js]
[C:\Users\Ciara\Rater-System\src\@angular\animations\browser]
[C:\Users\Ciara\Rater-System\src\@angular\animations\browser]
[C:\Users\Ciara\Rater-System\src\@angular\animations\browser.ts]
[C:\Users\Ciara\Rater-System\src\@angular\animations\browser.js]
[C:\Users\Ciara\Rater-System\src\@angular\animations\browser]
 @ ./node_modules/@angular/platform-browser/@angular/platform-browser/animations.es5.js 10:0-206
 @ ./src/app/app.module.ts
 @ ./src/main.ts
 @ multi ./src/main.ts

Reply:
Did you rebuild it?
Try a `npm install && ng build`

Reply:
Yes this is what comes out with ng build

Reply:
Ok.  That appears to me to be merge conflicts.  I'm not 100% sure but whenever I get errors that I have no idea what they mean it is usually merge conflicts lol